packages/terra-filter-view/src/_Dropdown.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Hookshot from 'terra-hookshot';
+import classNames from 'classnames/bind';
+import styles from './_Dropdown.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The content of the dropdown.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Whether the dropdown is presented above the target.
+   */
+  isAbove: PropTypes.bool,
+  /**
+   * Whether the dropdown is visible.
+   */
+  isEnabled: PropTypes.bool,
+  /**
+   * Callback function triggered when the dropdown resizes.
+   */
+  onResize: PropTypes.func.isRequired,
+  /**
+   * Callback function triggered for setting a ref to the dropdown.
+   */
+  refCallback: PropTypes.func.isRequired,
+  /**
+   * The attachment target.
+   */
+  target: PropTypes.shape({}).isRequired,
```

Comment:
Sneaky... You might be able to use `PropTypes.instanceOf(Element)` to avoid the lint error